feature/search/src/main/java/kr/co/cotton/feature/search/view/SearchSeriesView.kt

```diff
@@ -1,18 +1,75 @@
 package kr.co.cotton.feature.search.view
 
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
```

Comment:
🚫 **[ktlint]** <no-wildcard-imports> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wildcard import (cannot be auto-corrected)